assets/javascripts/job_next_previous.js

```diff
@@ -28,7 +34,14 @@ function setupJobNextPrevious() {
             },
             {targets: 2, render: renderBuild},
             {targets: 3, render: renderFinishTime},
-        ]
+        ],
+        initComplete: function (settings, json) {
+            // Seems an issue in case of displayStart is not an integer multiple of the pageLength
```

Comment:
hmm, wouldn't it still be easier to take your var page as displayStart = page * pageLength? I find changing the table in initComplete a little problematic - as initComplete is called after the table was drawn, meaning it would blink

Reply:
i didn't see blinking during initComplete called, from console no repeating draw shown.
test instance: http://10.67.18.160:3000/tests/72

Reply:
options can't be changed after initialized, i wonder to know a better way to pass value to option during in initialisation?

Reply:
I see. So we would have to calculate this earlier - which would break lazy load. 

Your test instance doesn't work for me and I'm in the middle of something else, so I can't test locally. So having to trust you that there is no blink :)

Reply:
Seems to work without blinking, also when 'Next & previous results' is the initial tab.